README.md

```diff
@@ -15,15 +15,12 @@ Please see the [CHANGELOG](CHANGELOG.md) for released & recent changes.
 
 ## Installation & Running
 
-We recommend installing `zulip-term` in a python virtual environment, something like the following:
-```
-[sudo] pip3 install virtualenv
-virtualenv /tmp/zt/
-. /tmp/zt/bin/activate
-pip3 install zulip-term
-```
+We recommend installing `zulip-term` in a new python virtual environment (venv); with the required python 3.4+, the following should work on most systems:
+1. `python3 -m venv zulip-terminal-venv` (creates a venv named `zulip-terminal-venv` in the current directory)
+2. `. zulip-terminal-venv/bin/activate` (activates the venv)
```

Comment:
here i think it should be `source zulip-terminal-venv/bin/activate`.

Reply:
No. @vishal-vardhan It is correct.

Reply:
`.` is POSIX-compliance, `source` is for bash-only (and bash-like shells). Nowadays it is fine to use `source` -- one minor advantage: it stands out more than `.` because it could happen that people might forget to select all the way to `.`.

Reply:
@amanagr  @rht sorry about that i forgot.

Reply:
@rht Is `source` sufficiently cross-shell/platform now? I can see the visual preference, but obviously cross-platform behavior would be preferable.

Reply:
Digging at zulip/zulip for a tie-breaker, but can't find any usage of it. IIRC it was recently replaced by `./executable.sh` instead of `. executable.sh`. Then found that https://virtualenv.pypa.io/en/latest/userguide/#activate-script uses `source` by default, but describes the caveats of using it in a paragraph afterward. TIL that `bin/activate` is only for bash-like shells. `csh`/`fish` users must source from a different file. Ergo, since saying `bin/activate` itself already assumes that users are on bash-like shells, there isn't any additional assumption to make when saying `source`. tl; dr: use `source`?

Reply:
@rht @amanagr So what do you think about just having '`source zulip-terminal-venv/bin/activate` (activates the venv, assuming a bash shell)' ?

Reply:
Yes, that's a safe assumption for people using a terminal client on laptops nowadays. Can't imagine anyone would run ZT on TempleOS, but if someone does, s/he probably knows enough to use `.` and a `venv/bin/activate.shellname`.

Reply:
I am fine with using `source`.

Reply:
OK; updating it now.